frontend/src/pages/learn/case-studies/fred.tsx

```diff
@@ -104,30 +108,107 @@ const Fred: FC = () => {
             </ListItem>
           ))}
         </List>
+        <AlertCard type="tip" className="mt-5">
+          <Trans
+            ns="learn/case-studies/fred"
+            i18nKey="key-takeaways.smart-tip"
+            components={{
+              a1: (
+                <a
+                  className="text-link-default underline visited:text-link-visited focus-visible:text-link-selected"
+                  href={t('key-takeaways.a1')}
+                />
+              ),
+              a2: (
+                <a
+                  className="text-link-default underline visited:text-link-visited focus-visible:text-link-selected"
+                  href={t('key-takeaways.a2')}
+                />
+              ),
+            }}
```

Comment:
Use MUI Link component and do the same in other places.
```suggestion
            components={{
              a1: <MuiLink href={t('key-takeaways.a1')} />,
              a2: <MuiLink href={t('key-takeaways.a2')} />,
            }}
```